src/assets/sass/protocol/components/_card.scss

```diff
@@ -77,13 +77,19 @@
         margin: $margin-xs 0 0;
     }
 
+    .mzp-c-card-cta {
+        font-weight: bold;
+        margin: $margin-xs 0 0;
+    }
+
     .mzp-c-card-block-link {
         border-radius: 2px;
         display: block;
 
         &:link,
         &:visited {
             color: $color-black;
+            border-bottom: 1px solid transparent;
```

Comment:
Should this be on `.mzp-c-card-title` and `.mzp-c-card-cta-text` instead of the block link? It should also be `2px` to match the hover state?

```css
.mzp-c-card-title,
.mzp-c-card-cta-text {
    border-bottom: 2px solid transparent;
}
```

Reply:
It's 1px to match what's on the cta link component. There's a transition that makes it look like it's growing.

And that transition means that it can't be put on a block level element :/ Personally, I'm not sure we should have custom underline effects with transitions. But since we do I'm trying to replicate how it was done.

Maybe it needs a mixin instead of being copied?

Reply:
I think that 1px transition may be a mistake tbh, I don’t think it’s intentional?

Reply:
Well, they should match. 

 @craigcook I know the discussion about the link underline effect was long and complicated. In https://github.com/mozilla/protocol/blob/master/src/assets/sass/protocol/base/elements/_links.scss#L28-L59 the transition is only on the colour and not the border-width. Should we remove the 1px border width? Increase it to 2px to match the hover state?

Reply:
Fwiw, I think I wrote this code originally. It should be 2px, I just forgot to update the before-hover value when updating based on feedback from our designers. I think it’s safe just update it across the board and save a long discussion 🙈

Reply:
Here’s the request to change it to 2px for transparency: https://github.com/mozilla/protocol/issues/137/